src/martialarts.cpp

```diff
@@ -47,11 +47,45 @@ static const matec_id tec_none( "tec_none" );
 
 namespace
 {
+generic_factory<weapon_category> weapon_category_factory( "mood_face" );
```

Comment:
This string is used in debug messages to describe the object type.
```suggestion
generic_factory<weapon_category> weapon_category_factory( "weapon category" );
```

Reply:
Oops thanks. I copied and pasted it from my other code and forgot to change that.